sandbox.go

```diff
@@ -747,6 +752,17 @@ func (eh epHeap) Less(i, j int) bool {
 	ci, _ := eh[i].getSandbox()
 	cj, _ := eh[j].getSandbox()
 
+	epi := eh[i]
+	epj := eh[j]
+
+	if epi.endpointInGWNetwork() {
+		return true
+	}
+
+	if epj.endpointInGWNetwork() {
+		return true
```

Comment:
Changed it. As we discussed for setting the GW we have to walk through the list of EPs and pick the first one with GW.  I will make that change in a separate PR.